packages/quantic/force-app/main/default/lwc/quanticBreadcrumbManager/quanticBreadcrumbManager.html

```diff
@@ -38,8 +38,8 @@
             </li>
           </template>
           <template for:each={categoryFacetBreadcrumbsValues} for:item="breadcrumbValue">
-            <li key={breadcrumbValue.facetId} class="slds-list_horizontal breadcrumb-manager__line">
-              <span class="slds-col breadcrumb-manager__field-name">{breadcrumbValue.field}{labels.colon} </span>
+            <li key={breadcrumbValue.facetId} class="slds-list_horizontal categoryFacetBreadcrumb__list breadcrumb-manager__line">
```

Comment:
The class name should be something like `breadcrumb-manager__category-facet-list`.